### ISSUE TITLE
  Feature request: notifications to admin

### ISSUE THREAD

**Issue by rogierl** [thorsten/phpMyFAQ#995]
Every division in our company has a group of users, who can only add questions, and one admin. The admins don't receive any notifications when a user adds an item. Is it possible to add a feature like this? For this to work, you should be able to create groups and assign an admin to every group. This is not possible right now (assigning an admin I mean).

Second feature request is about reviewing unanswered (on hold) questions: when one of the admins checks them, they see all the unanswered questions. They should only see the questions added by their users.

So in other words: is it possible to extend the user possibilities and add features like the above? This would add a lot of value to an already very useful system.

**Comment by thorsten** [comment]
I think this would require a matrix based authorization, right?

**Comment by rogierl** [comment]
I have no idea, I am sorry. My technical knowledge doesn't go this far.

**Comment by jesa43** [comment]
What about pinpointing a user in the group configuration, as an admin, and when checking whereto send the emails it adds sending the post to the admin as well. The second feature could be done if you add a group parameter for the user that is logged in and write it down in the DB for each questions and when the admin logs in and checks the questions on hold, use the group parameter to filter the questions.
